cookiecutter-django-ida/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py

```diff
@@ -26,15 +26,12 @@ def get_logger_config(log_dir='/var/tmp',
     """
 
     hostname = platform.node().split(".")[0]
-    syslog_format = (
-        "[service_variant={service_variant}]"
-        "[%(name)s][env:{logging_env}] %(levelname)s "
-        "[{hostname}  %(process)d] [%(filename)s:%(lineno)d] "
-        "- %(message)s"
-    ).format(
-        service_variant=service_variant,
-        logging_env=logging_env, hostname=hostname
-    )
+    syslog_format = (u"[service_variant={service_variant}]"
```

Comment:
Since everything is Python 3 now, I don't think we need the `u` prefix.

Reply:
There are also some indentation and linebreak changes that might not have been intentional.

Reply:
Latest changes should have resolved these two issues.